### PR TITLE
[BE] sort imports in `torch/__init__.py`

### PR DESCRIPTION
**Pull request by XuehaiPan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #127705
* #127704
* #128342
* #127707
* #127706
* #127710
* #127709
* __->__ #127708
* #127703

----

- Sort import via `usort`
- Change relative import `from . import xxx` to absolute import `from torch import xxx`

cc @albanD

### CODE REVIEW

File: torch/types.py [hunk 1]
```diff
@@ -7,10 +7,10 @@
 # Convenience aliases for common composite types that we need
 # to talk about in PyTorch
 
-_TensorOrTensors = Union[torch.Tensor, Sequence[torch.Tensor]]
+_TensorOrTensors = Union["torch.Tensor", Sequence["torch.Tensor"]]
 _TensorOrTensorsOrGradEdge = Union[
-    torch.Tensor,
-    Sequence[torch.Tensor],
+    "torch.Tensor",
```
Comment (Skylion007) [inline torch/types.py:12]: These should just be imported inside of a TYPE_CHECKING if block.

File: torch/__init__.py [hunk 2]
```diff
@@ -67,92 +76,98 @@ def _running_with_deploy():
...
+                res = kernel32.LoadLibraryW(dll)
+                if res is None:
+                    err = ctypes.WinError(ctypes.get_last_error())
+                    err.strerror += f' Error loading "{dll}" or one of its dependencies.'
+                    raise err
+
+        kernel32.SetErrorMode(prev_error_mode)
+
+    _load_dll_libraries()
+    del _load_dll_libraries
```
Comment (XuehaiPan) [inline torch/__init__.py:163]: Move the global initialization code to a function to do not expose the local variables to `torch.*` namespace.

**Review by ezyang — APPROVED** [review]

File: torch/__init__.py [hunk 3]
```diff
@@ -1457,17 +1472,25 @@ def _check_tensor_all(cond, message=None):  # noqa: F811
...
 __all__.extend(['e', 'pi', 'nan', 'inf', 'newaxis'])
 
 ################################################################################
 # Define Storage and Tensor classes
 ################################################################################
 
-from ._tensor import Tensor
-from torch import storage as storage
-from .storage import _StorageBase, TypedStorage, _LegacyStorage, UntypedStorage, _warn_typed_storage_removal
+from torch._tensor import Tensor  # usort: skip
```
Comment (albanD) [inline torch/__init__.py:1483]: Is there a lint for usort now?

Comment (XuehaiPan) [reply torch/__init__.py:1483]: `usort` is part of `ufmt`. The `ufmt` command runs `usort` and `black`.

**Pull request closed without merging**